lambda/examples/binary/index.js

```diff
@@ -0,0 +1,19 @@
+const fs = require("fs")
+const path = require("path")
+const crypto = require("crypto")
+
+const dataPath = path.join(__dirname, "index.zip")
+
+async function handler() {
+  const dataHash = crypto.createHash("md5")
```

Comment:
Am I correct that the content of this script has nothing to do with the hash computation in terraform and is just a sophisticated way to do something with the zip file attached? :)

Reply:
yes, exactly, needed a script that would actually do something with the binary, so I can check that it is there